README.md

```diff
@@ -89,9 +89,7 @@ After you're finished please remove all the comments and instructions!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🚀 [Live Demo] Coming soon...
-
-
+## 🚀 [Live Demo](https://riskmaris.github.io/My-Portfolio-Web-App./)
```

Comment:
- [ ] You must have wanted to write `https://riskmaris.github.io/My-Portfolio-Web-App/`, that is the working link.